styles.css

```diff
@@ -265,6 +265,220 @@ body {
   color: #091e42;
 }
 
+.uber-popup,
+.facebook-popup,
+.multi-popup,
+.tonic-popup {
+  position: absolute;
+  background-color: #fff;
+  border-radius: 16px;
+  width: 90%;
+  height: 120vh;
+  display: none;
+  z-index: 10;
+  border: 1px solid #ebecf0;
+  left: 5%;
+  top: 5%;
+}
+
+.header-section.blur {
+  filter: blur(6px);
+  height: 100vh;
+  z-index: 1;
+}
+
+.toolbar.dim {
+  filter: blur(6px);
+}
+
+#work.none {
+  display: none;
+}
+
+.uber-popup.show,
+.facebook-popup.show,
+.multi-popup.show,
+.tonic-popup.show {
+  display: flex;
+}
+
+.toggle-popup4,
+.toggle-popup3,
+.toggle-popup2,
+.toggle-popup {
+  margin-left: 98%;
+  width: 22px;
+  height: 22px;
+  border: none;
+  background-color: #fff;
+  cursor: pointer;
+}
```

Comment:
You have done great at implementing the popup to display on clicking the "see project" button. Could you ensure that the popup shows as shown on the Figma design? The popup is supposed to occupy the screen such that scrolling on the page does not scroll it away